client/src/app/page.tsx

```diff
@@ -29,45 +34,54 @@ export default function Page() {
 
   return (
     <div className="min-h-screen bg-[#0D0D0E] p-1">
-      <div className="grid h-full grid-cols-9 gap-1">
-        {/* left buffer */}
-        <div className="col-span-1"></div>
-        {/* Live Lounge Map - Left Section */}
-        <div className="relative col-span-3 flex flex-col gap-1">
-          <div className="col-span-3 h-full rounded-md bg-[#20222C] p-4">
-            <h1 className="p-4 text-2xl font-bold text-white">Lounge Map</h1>
-            <Map onPCClick={handlePCClick} />
-          </div>
-          {selectedPC ? (
-            <PCInfo pc={selectedPC} isOccupied={isOccupied} />
-          ) : (
+      <AuthStatus />
+      {isAuthenticated ? (
+        <div className="grid h-full grid-cols-9 gap-1">
```

Comment:
Getting to the point where this file is a bit bloated... we should consider factoring out the lounge home page as a component and the not logged in page as its own page (in case we ever decide to add more features to that page) and move them to separate files. This probably should've been done earlier but I think the more minimal page.tsx is the better